notdiamond/llms/request.py

```diff
@@ -178,6 +178,9 @@ def model_select(
     )
 
     try:
+        print(url)
```

Comment:
I think you might have forgotten to remove these print statements

Reply:
Hard to write software while holding a sick baby, oy. Removed, thank you for catching.